camera-core/src/main/java/com/stripe/android/camera/CameraPermissionCheckingActivity.kt

```diff
@@ -47,7 +47,7 @@ abstract class CameraPermissionCheckingActivity : AppCompatActivity() {
      * Check the camera permission, invokes [onCameraReady] upon permission grant,
      * invokes [onUserDeniedCameraPermission] otherwise.
      */
-    protected fun ensureCameraPermission() = when {
+    protected fun ensureCameraPermission(): Any = when {
```

Comment:
can we convert this to a `Unit` return value rather than `Any`? I think we would just need to make this change:

```
mainScope.launch {
    onCameraReady()
}.let { }
```